Documentation/ceph-examples.md

```diff
@@ -43,27 +41,37 @@ Settings for the operator are configured through environment variables on the op
 - `cluster-test.yaml`: Settings for a test cluster where redundancy is not configured. Requires only a single node.
-- `cluster-minimal.yaml`: Brings up a cluster with only one mon and a mgr so the Ceph dashboard can be used for the remaining cluster configuration.
+- `cluster-minimal.yaml`: Brings up a cluster with only one [ceph-mon](http://docs.ceph.com/docs/mimic/man/8/ceph-mon/) and a [ceph-mgr](http://docs.ceph.com/docs/mimic/mgr/) so the Ceph dashboard can be used for the remaining cluster configuration.
 
 See the [Cluster CRD](ceph-cluster-crd.md) topic for more details on the settings.
 
-## Storage Class
-The storage class is defined with a pool which defines the level of data redundancy:
-- `storageclass.yaml`: Replication of 3 for production scenarios. Requires at least three nodes.
-- `storageclass-ec.yaml`: Configures erasure coding for data durability rather than replication. See the [Erasure coding](ceph-pool-crd.md#erasure-coded) documentation for more details. Requires at least three nodes.
-- `storageclass-test.yaml`: Replication of 1 for test scenarios. Requires only a single node.
+## Setting up consummable storage
+
+Now we are ready to setup [block](https://ceph.com/ceph-storage/block-storage/), [shared filesystem](https://ceph.com/ceph-storage/file-system/) or [object storage](https://ceph.com/ceph-storage/object-storage/) in the rook ceph cluster. These kinds of storage are respectively referred to as CephBlockPool, CephFilesystem and CephObjectStore in the spec files.
+### Block Devices
+
+Ceph can provide raw block device volumes to pods. Each example below sets up a storage class which can then be used to provision a block device in kubernetes pods. The storage class is defined with [a pool](http://docs.ceph.com/docs/jewel/rados/operations/pools/) which defines the level of data redundancy in ceph:
+
+- `storageclass.yaml`: This example illustrates replication of 3 for production scenarios and requires at least three nodes. Your data is replicated on three different kubernetes worker nodes and intermittent or long-lasting single node failures will not result in data unavailability or loss.
+- `storageclass-ec.yaml`: Configures erasure coding for data durability rather than replication. [Ceph's erasure coding](http://docs.ceph.com/docs/mimic/rados/operations/erasure-code/) is more efficient than replication so you can get high reliability without the 3x replication cost of the preceding example (but at the cost of higher computational encoding and decoding costs on the worker nodes). Erasure coding requires at least three nodes. See the [Erasure coding](ceph-pool-crd.md#erasure-coded) documentation for more details. 
+- `storageclass-test.yaml`: Replication of 1 for test scenarios and it requires only a single node. Do not use this for applications that store valuable data or have high-availability storage requirements, since a single node failure can make data unavailable.
```

Comment:
shall we mention data loss instead of just that the data would be unavailable?

Reply:
Changed to data loss.